imports/ui/components/Profile/Dialog.jsx

```diff
@@ -25,10 +25,21 @@ const styles = theme => ({
   }
 });
 class DialogSelect extends React.Component {
-
+  state = { backEndLevel: "", frontEndLevel: "", dataLevel: "" };

```

Comment:
we should put the state in profile page, as it contains all the core logics for interaction